source/guide_neos.rst

```diff
@@ -0,0 +1,155 @@
+.. highlight:: console
+
+.. sidebar:: Logo
+  
+  .. image:: _static/images/neos.png 
+      :align: center
+
+#########
+Neos
+#########
+
+Neos_ is an open source Content Application Platform based on its own PHP framework Flow and distributed under the GPLv3 licence.
+
+Neos (formerly TYPO3 Neos) was released for the first time in 2013. It is maintained by the Neos-Team and Contributors.
+
+
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of 
+
+  * PHP_
+  * MySQL_ 
+  * domains_
+
+Prerequisites
+=============
+
+We're using PHP_ in the stable version 7.1: 
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version show php
+ Using 'PHP' version: '7.1'
+ [isabell@stardust ~]$
+
+.. include:: includes/my-print-defaults.rst
+
+Your website domain needs to be setup:
+
+.. include:: includes/web-domain-list.rst
+
+You need to install the dependency manager Composer (if you don’t have it already) to create your Neos project:
```

Comment:
We will provide composer globally in a few days, I'll keep you postet on this one.

Reply:
Thanks Moritz. I'll update the guide when it's ready.

Reply:
There you go: https://manual.uberspace.de/en/changelog.html#v7-1-2